include/oneapi/dpl/pstl/hetero/dpcpp/sycl_defs.h

```diff
@@ -130,6 +130,8 @@ using __maximum = sycl::maximum<_T>;
 template <typename _T = void>
 using __minimum = sycl::minimum<_T>;
 
+template <typename _T = void>
+using __bit_or = sycl::bit_or<_T>;
```

Comment:
Described at https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#table.identities

Reply:
Test evidence for complier icx 2021.1.2 (whene group operations isn't present) ): https://godbolt.org/z/PcnYT4vqe